hera_mc/cm_handling.py

```diff
@@ -387,23 +421,25 @@ def get_part_connection_dossier(self, hpn_list, rev, port, at_date, exact_match=
         Parameters
         -----------
         hpn_list:  the input hera part number [list of strings] (whole or first part thereof)
-        rev:  specific revision(s) or category(ies) ('LAST', 'ACTIVE', 'ALL', 'FULL')
+        rev:  specific revision(s) or category(ies) ('LAST', 'ACTIVE', 'ALL', 'FULL', specific)
               if list, must match length of hpn_list
         port:  a specifiable port name [string, not a list],  default is 'all'
-        at_date: reference date of dossier [anything get_astropytime can handle]
+        at_date: reference date of dossier, only used if rev==ACTIVE (and for now FULLY_CONNECTED...)
```

Comment:
is `FULLY_CONNECTED` the same as `FULL` mentioned 3 lines above? If so, use the same name here.